code-challenges.test.js

```diff
@@ -38,9 +61,19 @@ const object3 = { number: -7 }
 
 // b) Create the function that makes the test pass.
 
+const even3 = (object) => {
+    return object.number % 3 ===0 ? `${object.number} is divisible by three` : `${object.number} is not divisible by three`
+}
+console.log(even3(object1));
+console.log(even3(object2));
+console.log(even3(object3));
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
-// Pseudo code:
+// Pseudo code: 
+// input: array that contains strings
+// output : string with the first letter capitilized
+// process: Will use indexing and toUpperCase() to change the string
```

Comment:
There is no mention of `.map` or `.replace` and how they work in your pseudo code!